licenses/third_party.md

```diff
@@ -111,7 +111,6 @@ run `task generate:third-party-licenses-list`
 | github.com/gonvenience/ytbx | [MIT](https://github.com/gonvenience/ytbx/blob/v1.4.4/LICENSE) |
 | github.com/google/btree | [Apache-2.0](https://github.com/google/btree/blob/v1.1.3/LICENSE) |
 | github.com/google/cel-go | [Apache-2.0](https://github.com/google/cel-go/blob/v0.17.8/LICENSE) |
-| github.com/google/cel-go | [BSD-3-Clause](https://github.com/google/cel-go/blob/v0.17.8/LICENSE) |
```

Comment:
You might need to rebase to get this to go away? I bumped the version of go-licenses recently.

Reply:
Oh weird, it's actually be cause I was in an old nix-shell and `direnv` didn't pick up the change. Just exited the shell and re-entered and there's the `go-licenses` change.